docs/source/release/v6.3.0/diffraction.rst

```diff
@@ -47,7 +47,7 @@ Engineering Diffraction
 -----------------------
 New features
 ############
-- Now support texture grouping (10 groups per bank) for ENGIN-X in the :ref:`Engineering Diffraction interface<Engineering_Diffraction-ref>`. Note this involved changes to the bankID log values saved with focused data, so this means the UI will not load in previously focused .nxs files.
+- Now support two texture grouping schemes (10 groups per bank and 15 groups per bank) for ENGIN-X in the :ref:`Engineering Diffraction interface<Engineering_Diffraction-ref>`. Note this involved changes to the bankID log values saved with focused data, so this means the UI will not load in previously focused .nxs files.
```

Comment:
can we include the names of the two texture grouping schemes in the release note? I'm a little confused as this PR concerns Texture30 option which I can't easily relate to the description here (10 and 15 groups per bank are mentioned).